CRM/Core/Form.php

```diff
@@ -2653,6 +2653,25 @@ public function getCurrency($submittedValues = []) {
     return $currency;
   }
 
+  /**
+   * Get the submitted value of a form field.
+   *
+   * @param array $fieldName
+   *   Form field name.
+   *
+   * @return string
+   *   Submitted value of the form field.
+   */
+  public function getSubmittedValue ($fieldName) {
```

Comment:
Why is there a space here between function name and (

Reply:
Fixed.